src/plugins/vis_default_editor/public/components/controls/date_ranges.tsx

```diff
@@ -141,7 +141,10 @@ function DateRangesParamEditor({
     <EuiFormRow compressed fullWidth>
       <>
         <EuiText size="xs">
-          <EuiLink href={services.docLinks.links.date.dateMath} target="_blank">
+          <EuiLink
+            href={services.docLinks.links.opensearch.aggregations.bucket.range}
```

Comment:
is this the right link? bucket aggregations are a different thing.

Reply:
Not so sure, I was thinking to use : https://opensearch.org/docs/latest/opensearch/units/ but it doesn't contains enough info related date `range`.  Instead, https://opensearch.org/docs/latest/opensearch/bucket-agg/#range-date_range-ip_range, contains more info about `date_range`. Open to any inputs